### PR TITLE
chore: bump alloy, reth, revm, and eip3074-instructions

### PR DESCRIPTION
**Pull request by emostov**
relies on https://github.com/paradigmxyz/eip3074-instructions/pull/2

TODO
- [ ] point eip3074-instructions back to primary repo once https://github.com/paradigmxyz/eip3074-instructions/pull/2
 is merged

### CODE REVIEW

**Comment by emostov** [comment]
I think this supersedes #84

File: Cargo.toml [hunk 1]
```diff
@@ -44,37 +44,34 @@ incremental = false
...
-reth-tracing = { git = "https://github.com/paradigmxyz/reth.git", rev = "40d8ac0" }
+reth = { git = "https://github.com/paradigmxyz/reth.git", rev = "539b4e4", features = ["optimism"] }
+reth-chainspec = { git = "https://github.com/paradigmxyz/reth.git", rev = "539b4e4" }
+reth-cli-util = { git = "https://github.com/paradigmxyz/reth.git", rev = "539b4e4" }
+reth-node-api = { git = "https://github.com/paradigmxyz/reth.git", rev = "539b4e4" }
+reth-node-builder = { git = "https://github.com/paradigmxyz/reth.git", rev = "539b4e4" }
+reth-node-core = { git = "https://github.com/paradigmxyz/reth.git", rev = "539b4e4" }
+reth-node-optimism = { git = "https://github.com/paradigmxyz/reth.git", rev = "539b4e4" }
+reth-primitives = { git = "https://github.com/paradigmxyz/reth.git", rev = "539b4e4" }
+reth-tracing = { git = "https://github.com/paradigmxyz/reth.git", rev = "539b4e4" }
```
Comment (mattsse) [inline Cargo.toml:68]: lets start using tags "v1.0.2"

File: Cargo.toml [hunk 2]
```diff
@@ -44,37 +44,34 @@ incremental = false
...
 
 # misc
 clap = "4"
 eyre = "0.6.12"
 tracing = "0.1.0"
-eip3074-instructions = { git = "https://github.com/paradigmxyz/eip3074-instructions.git", rev = "a9022e4" }
+eip3074-instructions = { git = "https://github.com/emostov/eip3074-instructions.git", branch = "bump-revm" }
```
Comment (mattsse) [inline Cargo.toml:74]: could you perhaps also bump the eip3074 instructions repo?

**Review by mattsse — CHANGES_REQUESTED** [review]
tysm for this, this has been on my list for a bit -.-

File: Cargo.toml (hunk 2 above)
Comment (emostov) [reply Cargo.toml:74]: pr here https://github.com/paradigmxyz/eip3074-instructions/pull/2

Comment (emostov) [reply Cargo.toml:74]: actually looks like i need to bump msrv here and in eip3074 prs

File: Cargo.toml (hunk 1 above)
Comment (emostov) [reply Cargo.toml]: [e5dc132](https://github.com/paradigmxyz/alphanet/pull/88/commits/e5dc1329f520abcaaf9b9a4d5a9c98fcc6d6f82e)

**Review by mattsse — APPROVED** [review]
amazing!

**Pull request merged**